Automattic/simplenote-android
# Update GitHub Templates

### Fix
These changes include:
* Update the issue templates in the `.github/ISSUE_TEMPLATE` directory
* Add the `.github/PULL_REQUEST_TEMPLATE.md` file for pull requests
* Remove the legacy `.github/ISSUE_TEMPLATE.md` file for issues

### Test
1. Open the `---bug-report.md`, `---feature-request.md`, and `PULL_REQUEST_TEMPLATE.md` files.
2. Verify instructions are displayed.

### Review
Only one developer is required to review these changes, but anyone can perform the review.
